src/linux/cpu.rs

```diff
@@ -10,12 +10,16 @@ use std::time::{Duration, Instant};
 use crate::sys::utils::to_u64;
 use crate::{CpuExt, CpuRefreshKind};
 
+#[cfg(feature = "serde")]
+use serde::Serialize;
+
 macro_rules! to_str {
     ($e:expr) => {
         unsafe { std::str::from_utf8_unchecked($e) }
     };
 }
 
+#[cfg_attr(feature = "serde", derive(Serialize))]
```

Comment:
This is a private type, there is no point into making it serializable.

Reply:
I can remove serialization of struct a that don't need to be serialized.